src/ImageRunModal.jsx

```diff
@@ -1070,7 +1070,7 @@ export class ImageRunModal extends React.Component {
                                   <Popover aria-label={_("Health failure check action help")}
                                       enableFlip
                                       bodyContent={_("Action to take once the container transitions to an unhealthy state.")}>
-                                      <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                                      <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1322-20230703-042644-64a79ae1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)